components/Container/Container.js

```diff
@@ -1,18 +0,0 @@
-import React, {Component} from 'react';
-
-import styles from './Container.css';
-
-export default class Container extends Component {
-  render() {
-    return (
-      <div className={[styles.root, this.props.className].join(' ')}>
-        {this.props.children}
-      </div>
-    );
-  }
-}
-
-Container.propTypes = {
-  children: React.PropTypes.any,
-  className: React.PropTypes.string
-};
```

Comment:
`<Container>` was always a bit of a hack. Now we're using [`<BodyClassName/>`](https://github.com/PactCoffee/react-body-classname) (which makes that stuff waaaay nicer), we don't need container anymore.